test/mailers/passsword_mailer_test.rb

```diff
@@ -0,0 +1,12 @@
+require "test_helper"
+
```

Comment:
Are these a separate class of tests or can be added in spec folder?

Reply:
these are autogenerated files from mailer. not sure where to place them